static/include/services/apis/overrides/protos/slam.PointCloudMapFull.md

```diff
@@ -0,0 +1 @@
+`PointCloudMapFull` concatenates the streaming responses from `PointCloudMap` into a full point cloud.
```

Comment:
I am not 100% sure about how to word this--I defer to @JohnN193 for semantic correctness, though I realize this does come from the comment in RDK so maybe I'm being overly cautious.

Reply:
Verbatim scrape from upstream. Whatever is found [there](https://pkg.go.dev/go.viam.com/rdk@v0.27.1/services/slam#PointCloudMapFull) will be replicated here.